di-ipv-queue-stub/enqueue-event/package.json

```diff
@@ -8,6 +8,7 @@
         "@babel/core": "^7.21.8",
         "@babel/preset-env": "^7.21.5",
         "@babel/preset-typescript": "^7.21.5",
+        "@types/node": "^20.2.5",
         "aws-lambda": "^1.0.7",
         "babel-jest": "^29.5.0",
         "jest": "^29.2.1"
```

Comment:
Not a big deal but if we're not running tests for this can bin this down to line 28 (save for @types/node)